ebms-provider/src/main/kotlin/no/nav/emottak/ebms/processing/ProcessingService.kt

```diff
@@ -81,6 +85,22 @@ class ProcessingService(private val httpClient: PayloadProcessingClient) {
     }
 
     private fun acknowledgment(acknowledgment: Acknowledgment) {
```

Comment:
Denne koden er ikke implementert enda, og vil sannsynligvis skrives om ganske mye, men denne funksjonen er ment for prosessering av innkommende `acknowledgment`. Koden som produserer acknowledgment for en gitt payload-melding skjer i routen til async flyten: https://github.com/navikt/ebxml-processor/blob/6a0ce758d3526d019ea49afd67d20ab17349f1f8/ebms-provider/src/main/kotlin/no/nav/emottak/ebms/Routes.kt#L291-L294

Det er denne produserte responsen som skal legges på topicen.